README.md

```diff
@@ -341,31 +287,13 @@ If you don't want to pass GitHub API token to this action, it's still OK.
     # Set auto-push to false since GitHub API token is not given
     auto-push: false
 # Push gh-pages branch by yourself
-- name: Push benchmark result
+- name: Publish to github pages without token
   run: git push 'https://you:${{ secrets.GITHUB_TOKEN }}@github.com/you/repo-name.git' gh-pages:gh-pages
 ```
 
 Please add a step to push the branch to the remote.
 
 
-### Tool specific setup
```

Comment:
Same. These have value to users.

Reply:
kept 👍

Reply:
This section is repeated twice. 
- https://github.com/benchmark-action/github-action-benchmark?tab=readme-ov-file#commit-comment:~:text=This%20action%20currently%20supports%20the%20following%20tools%3A
- and here https://github.com/benchmark-action/github-action-benchmark?tab=readme-ov-file#tool-specific-setup

I updated the second one and removed the first one.